### ISSUE TITLE
Cadence running on ARM cpus

### ISSUE THREAD

**Issue by skyman9** [falkTX/Cadence#124]
is there any chance of running cadence on a ARM cpu/ and the other applications fro kxstudio

**Comment by falkTX** [comment]
is there anything stopping you from doing so?
I had cadence and carla running on a raspberry pi not so long ago.

just get the source and build it.

**Comment by skyman9** [comment]
hello, im faulty new to linux, building from source code sounds very complex and intimidating or maybe it is because i don't know much about doing that. 
its good to know that its possible to run it, but I'm not sure where to start with all that, is it complicated process,
i have other programs i wanted to get running as well so if i learn how to do it, then i guess i would not have to miss out on leaving my programs behind

**Comment by falkTX** [comment]
if you're new to linux you should not be using development boards like raspberry pi.
use a normal pc instead.

**Comment by skyman9** [comment]
i hear that, I've used linux before, so i have some experience with the OS, but not with compiling, i am a tech nerd but with linux it really makes me realize I'm a noob lol, 
reason is now that I'm making a custom device for audio mastering, the main components it uses is a ARM based cpu, i know i can use a intel based chip to avoid all of this, but i really want to use the ARM cpu this time around, plus intel is too costly which is not what I'm looking for.

**Comment by Rippert** [comment]
OK - this is a really old issue, but it is still open.
I have successfully compiled Cadence on the latest raspberian (both lite and full), but it always complains about there being no DBUS to talk to jack with. Yet Gladish, which also uses DBUS works. Actually, Catia works fine, but Cadence and Claudia do not work. Claudia won't even open, It just gives a dialog saying there is no DBUS.
All I did was download your latest release code and used the configure errors to find the right dependencies to add.
Any suggestions?

Thanks (especially for KXStudio which I use on my MacBook)
-Ted

**Comment by falkTX** [comment]
Catia can start without DBus, that's why it works for you (but some features are then disabled)

@Rippert  What ARM distro are you using?
Did you build the latest Cadence from git?
Do you have python3-dbus.mainloop.pyqt5 package installed?
If yes, please paste here the contents of the package. (you can do so by using dpkg -L <pkg>)

**Comment by Rippert** [comment]
python3-dbus.mainloop.pyqt5 was not installed. I installed it and now Cadence is able to control Jack, and Claudia mostly works. I did notice that the "Arrange" menu item is greyed out in both Claudia and Catia. That's a useful item, any suggestions?

To answer your other questions:
Cadence was downloaded from git - top of git log:
`commit 99b39989a010fae1acbce07af002b15e56a1d161
Author: falkTX <falktx@gmail.com>
Date:   Thu Mar 22 00:58:48 2018 +0100`

cat /etc/rpi-issue
`Raspberry Pi reference 2018-03-13
Generated using pi-gen, https://github.com/RPi-Distro/pi-gen, 00013d7972122d1304aacda8fff5098f073ceb43, stage5`

uname -a
`Linux rpi3 4.14.30-v7+ #1102 SMP Mon Mar 26 16:45:49 BST 2018 armv7l GNU/Linux`

cat /etc/os-release
`PRETTY_NAME="Raspbian GNU/Linux 9 (stretch)"
NAME="Raspbian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"`


So, basically the latest raspbian with the latest apt-get upgrade.

Thanks for the help and the software,
-Ted
